### PR TITLE
pages: add missing info-text elements in "Company" step

### PR DESCRIPTION
**Pull request by maceksimon**
To add two missing elements in the "Company" step of challenge registration:

* Add strings to translations
* Add element to `src/components/form/FormFieldSelectTable.vue` if variant is `company`
* Add element to `src/pages/RegisterChallengePage.vue` to the navigation section of "Company" step
* Add tests

### CODE REVIEW

File: src/components/form/FormFieldSelectTable.vue [hunk 1]
```diff
@@ -348,6 +348,19 @@ export default defineComponent({
...
     </q-field>
+    <div
+      v-if="variant === 'company'"
+      class="text-caption text-grey-7 q-mt-sm"
+      data-cy="form-select-table-user-note"
+    >
+      <span
+        v-html="
+          $t('form.company.textUserExperience', {
+            email: 'kontakt@dopracenakole.cz',
```
Comment (tmszi) [inline src/components/form/FormFieldSelectTable.vue:359]: E -mail should be obtained from a global config variable `contactEmail` https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/595c15d657ed3135fe15b5e422e85a0ce4f4fde9/ride_to_work_by_bike_config.toml#L20.

File: test/cypress/e2e/register_challenge.spec.cy.js [hunk 2]
```diff
@@ -442,6 +442,14 @@ describe('Register Challenge page', () => {
             .find('.q-radio')
             .first()
             .click();
+          cy.dataCy('step-4-info').should('be.visible');
```
Comment (tmszi) [inline test/cypress/e2e/register_challenge.spec.cy.js:445]: Could you test presence of div element with text _Have you noticed an incorrect or duplicate record? Please email us at [kontakt@dopracenakole.cz](mailto:kontakt@dopracenakole.cz)._ also, and add comment for these two cases, please?

File: src/components/form/FormFieldSelectTable.vue (hunk 1 above)
Comment (maceksimon) [reply src/components/form/FormFieldSelectTable.vue]: Yes, added contact address from config.

File: test/cypress/e2e/register_challenge.spec.cy.js (hunk 2 above)
Comment (maceksimon) [reply test/cypress/e2e/register_challenge.spec.cy.js:445]: See updated test in `FormFieldSelectTable.cy`.
https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/303/files#diff-e28d5236d1e6eedadb0f92fb28d5168b81ccc4512956910c2ea03b792403bde3R90
I think the component is better suited for testing this as the e2e test is already very long.

**Comment by tmszi** [comment]
Better with Cy custom command for striping HTML tags from text string https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/303/commits/d72d34e54b4fc4928052a129b0a9f9b98f96e1b9.

**Pull request merged**